editor/editor_audio_buses.cpp

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
It looks like your IDE added an unnecessary BOM (Byte Order Mark) here. Could you amend the commit to remove it?

Reply:
The BOM should be removed. Sorry about that, I'm not sure how that got there.